repair/repair.cc

```diff
@@ -1214,8 +1233,11 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
                 }
             }
             if (!errors.empty()) {
-                return make_exception_future<>(std::runtime_error(format("{}", errors)));
+                auto ex = std::runtime_error(format("{}", errors));
```

Comment:
this can be simplified a bit with `auto ex = std::make_exception_ptr(std::runtime_error(format("{}", errors)));`
and then you should have no problem to `return make_exception_future<>(std::move(ex));` below.